### ISSUE TITLE
supporting date intervals as eventDate

### ISSUE THREAD

**Issue by damianooldoni** [iobis/obis-issues#166]
I am wondering how OBIS handle date intervals in DwC field `eventDate`. Is there documentation about it available?
As I am mostly working with GBIF, I know this is quite an issue there (https://github.com/gbif/gbif-api/issues/4). Thanks in advance!

**Comment by pieterprovoost** [comment]
@damianooldoni Yes, we support ISO 8601 date ranges, but in many cases (such date filtering) we only take into account the midpoint of the interval. We intend to make better use of start and end dates in the future. The API does return the start, mid, and end dates as unix time: https://api.obis.org/occurrence/fd425d8f-a9d7-4a1d-9c40-e40b91fbc3b4

**Comment by damianooldoni** [comment]
Nice, thanks! A little extra question. In the link provided the time interval is within the same year. I suppose nothing changes if the interval spans multiple years. But which is the value of the field `date_year` in such case?

**Comment by pieterprovoost** [comment]
In that case `date_year` will be the year of the midpoint, see https://api.obis.org/occurrence/c9f89192-828f-420b-84fe-fa0c9eda14f5 for example. Suggestions for improvement are welcome!

**Comment by damianooldoni** [comment]
thanks @pieterprovoost. Nice!
